lib/appdirs/__init__.py

```diff
@@ -0,0 +1 @@
+from .appdirs import user_cache_dir
```

Comment:
Very good point. To manage and to upgrade packages in our lib would be straightforward